src/DependencyInjection/Compiler/MenuBuilderPass.php

```diff
@@ -12,9 +12,9 @@
  *
  * @internal
  */
-class MenuBuilderPass implements CompilerPassInterface
+final class MenuBuilderPass implements CompilerPassInterface
```

Comment:
please tag them as `@final` in a 2.x release to be safe.

Reply:
Which branch is intended to target 2.x release? Currently, there are only 2: [`v3` and `master`](https://github.com/KnpLabs/KnpMenuBundle/branches).

Reply:
@phansys master

Reply:
Then, shouldn't this PR target `v3`?

Reply:
No, v3 is just a temporary branch where this PR is already merged, useful to do a practical test.